ignite/pkg/cosmosfaucet/http_faucet.go

```diff
@@ -69,10 +69,13 @@ type FaucetInfoResponse struct {
 }
 
 func (f Faucet) faucetInfoHandler(w http.ResponseWriter, _ *http.Request) {
-	xhttp.ResponseJSON(w, http.StatusOK, FaucetInfoResponse{
+	err := xhttp.ResponseJSON(w, http.StatusOK, FaucetInfoResponse{
 		IsAFaucet: true,
 		ChainID:   f.chainID,
 	})
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I think is "safe" to omit the `panic` here, an error response will be returned if response marshalling fails. A panic might not be ideal here and in the other faucet handlers because it would highly likely take down the faucet process.